app/frontend/.env.development

```diff
@@ -1,3 +1,4 @@
+IS_BETA_ENABLED=false
```

Comment:
I think because of how Create React App needs it [as referenced here](https://create-react-app.dev/docs/adding-custom-environment-variables/), this needs to be prefixed with `REACT_APP_` for it to work.